frontend/packages/network-attachment-definition-plugin/src/flags/check-nad-permissions.ts

```diff
@@ -12,6 +13,7 @@ export const checkNadPermissions = async (setFeatureFlag: SetFeatureFlag) => {
       group: NetworkAttachmentDefinitionModel.apiGroup,
       resource: NetworkAttachmentDefinitionModel.plural,
       verb: 'list',
+      namespace: getActiveNamespace(),
```

Comment:
does not make sense to me, the bug is about removing the unneeded check, not changing it.

Reply:
thanks :) 
made some changes to the initial comment after the last iteration that should follow your tips